scapy/contrib/pnio_dcp.py

```diff
@@ -234,7 +234,7 @@ class DCPManufacturerSpecificBlock(Packet):
         ByteEnumField("option", 2, DCP_OPTIONS),
         MultiEnumField("sub_option", 1, DCP_SUBOPTIONS, fmt='B',
                        depends_on=lambda p: p.option),
-        FieldLenField("dcp_block_length", None),  # TODO
+        FieldLenField("dcp_block_length", None),
```

Comment:
This won't do much though :/

Reply:
Hi Gabriel, 
sure this has no effect, but gueou asked me to remove the TODO tags. 
Best

Reply:
I think he meant fixing them rather than just removing the tags..